app/models/concerns/audit_extensions.rb

```diff
@@ -65,6 +67,8 @@ def self.auditable_type_complete_values
     scoped_search :relation => :search_hostgroups, :on => :name, :complete_value => true, :rename => :hostgroup, :only_explicit => true
     scoped_search :relation => :search_hostgroups, :on => :title, :complete_value => true, :rename => :hostgroup_title, :only_explicit => true
     scoped_search :relation => :search_users, :on => :login, :complete_value => true, :rename => :user, :only_explicit => true
+    scoped_search :relation => :search_nics, :on => :name, :complete_value => true, :rename => :interface, :only_explicit => true
```

Comment:
`interface = host.example.com` is slightly confusing, maybe better name would be interface_hostname or interface_fqdn?